src/styles/button.scss

```diff
@@ -0,0 +1,24 @@
+.card__button {
+  width: 166px;
+  height: 40px;
+  padding: 12px 69px 12px 70px;
```

Comment:
If you've hardcoded width and height padding is redundant. It should work without it. Or vice versa